docs/source/public.rst

```diff
@@ -1,8 +1,8 @@
 Public & users
 ==============
 
-Models
-------
+Models used
+-----------
 
 .. automodule:: purchasing.users.models
```

Comment:
This should be switched to :py:class: reference tags, like in the Beacon or Scout docs